### PR TITLE
#20 about us location section

### PR DESCRIPTION
**Pull request by adamswonder**
### About us location section [⚡#20](https://github.com/italanta/elewa-group/issues/20)

# Summary

This feature shows the physical location of the organization and a link to get directions using angular google maps. It makes it easier for one to effectively map out the organization's address using geolocation.

To achieve this, we needed to:

```
- Find the best map API/service which would help pinpoint the organization's precise location
- Apply for an API_KEY and Integrate it in the page section to give the location and/or directions
- Provide a marker making it visible on the page section so users can notice it easily
```
```
- Used placeholder for the map

`<div class="location-main-flex">
  <!-- Used an image placeholder for the map. Append the key and comment out the img tag -->
  <div class="map">
    <google-map height="400px" width="400px" [options]="mapOptions">
      <map-marker [position]="markerPositions"></map-marker>
    </google-map>
  </div>

  <div class="side-text">
    <div>
      <h1>How to get there</h1>

      <p>
        The secret to successful consectetur adipisicing elit. Et error sunt rem
        quia porro quidem praesentium obcaecati quibusdam aliquam. Laudantium
        eveniet libero enim explicabo ratione voluptate illum eum, ipsam
        repellat!
      </p>

      <elewa-group-button-with-animation
        [message]="'Get directions'"
        [mode]="'dark'"
        [action]="'/'"
      ></elewa-group-button-with-animation>
    </div>
  </div>
</div>
...`

```

Fixes [#20 ]

## Type of change

- [x] New feature (non-breaking change which adds functionality)

# Overview
## Screenshot
desktop
mobile

## Live Demo

https://user-images.githubusercontent.com/108331299/218426189-bbc35223-fea1-4dbd-9867-4c0e48df6540.mp4

# Checklist:
- [x] Our code follows the style guidelines of this project
- [x] We have performed a self-review of our code
- [x] We have commented on our code, particularly in hard-to-understand areas
- [x] Our changes generate no new warnings

-------------------------------
CC: @JamesMGithire

### CODE REVIEW

**Review by JamesMGithire — COMMENTED** [review]
Code changes are okay🚀

File: libs/pages/elewa/about-us/src/lib/components/elewa-about-us-location-section/elewa-about-us-location-section.component.css [hunk 1]
```diff
@@ -0,0 +1,72 @@
+.location-main-flex{
+    margin: auto;
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+    justify-content: space-evenly;
+    background-color: var(--elewa-group-website-color-card);
+    padding: 32px 0;
+}
+
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/about-us/src/lib/components/elewa-about-us-location-section/elewa-about-us-location-section.component.css:10]: This should be a .scss file not css

File: libs/pages/elewa/about-us/src/lib/components/elewa-about-us-location-section/elewa-about-us-location-section.component.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  options: google.maps.MapOptions = {
+    mapTypeId: 'hybrid',
+    zoomControl: false,
+    scrollwheel: false,
+    disableDoubleClickZoom: true,
+  };
+ 
+  ngOnInit() {
+    navigator.geolocation.getCurrentPosition((position) => {
+      this.center = {
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/about-us/src/lib/components/elewa-about-us-location-section/elewa-about-us-location-section.component.ts:27]: Lat and long points for ELewa location

-1.256132162858307, 36.80501589843353

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
You need to add a marker on the map with the provided Geo-points

**Review by JamesMGithire — COMMENTED** [review]
Completed chaneges.

**Review by IanOdhiambo9 — APPROVED** [review]

**Pull request merged**